tensorflow/java/src/main/java/org/tensorflow/Output.java

```diff
@@ -20,8 +20,11 @@
  *
  * <p>An Output is a symbolic handle to a tensor. The value of the Tensor is computed by executing
  * the {@link Operation} in a {@link Session}.
+ *
+ * <p>By implementing the {@link Input} interface, instances of this class could also be passed
+ * directly in input to an operation wrapper when using the Java Ops API.
```

Comment:
Again, I'd suggest avoiding the term "Java Ops API", instead just say "directly in input to an operation"?